Makefile

```diff
@@ -102,7 +108,10 @@ docker-build:
 	docker build $(BUILD_PROXY) -t $(IMAGE_BUILDER) . 
 
 clean:
-	rm -rf "$(WORKDIR)/rootfs" "$(WORKDIR)/linux" "$(WORKDIR)/img"
+	sudo rm -rf "$(WORKDIR)/rootfs"
+	rm -rf "$(WORKDIR)/linux"
+	rm -rf "$(WORKDIR)/img"
```

Comment:
Q. should this be ```sudo``` too?

Reply:
I am adding just to linux because the rootfs files may be owned by root (it is a a rootfs). I dont think the rest are need to have sudo.